test/unit/external_return_label_request_test.rb

```diff
@@ -3,121 +3,164 @@
 class ExternalReturnLabelRequestTest < ActiveSupport::TestCase
   include ActiveShipping::Test::Fixtures
 
-  def setup
+  setup do
     @external_request_label_req =
-      ExternalReturnLabelRequest.from_hash ({
-        :customer_name => "Test Customer",
-        :customer_address1 => "122 Hudson St.",
-        :customer_city => "New York",
-        :customer_state => "NY",
-        :customer_zipcode => "10013",
-        :label_format => "No Instructions",
-        :label_definition => "4X6",
-        :service_type_code => "044",
-        :merchant_account_id => "12345",
-        :mid => "12345678",
-        :call_center_or_self_service => "Customer",
-        :address_override_notification => "true"
-      })
-
+      ExternalReturnLabelRequest.from_hash(
+        customer_name: "Test Customer",
+        customer_address1: "122 Hudson St.",
+        customer_city: "New York",
+        customer_state: "NY",
+        customer_zipcode: "10013",
+        label_format: "No Instructions",
+        label_definition: "4X6",
+        service_type_code: "044",
+        merchant_account_id: "12345",
+        mid: "12345678",
+        call_center_or_self_service: "Customer",
+        address_override_notification: "true",
+      )
+    @email = "no-reply@example.com"
+    @invalid_email = "not_a_valid_email"
   end
 
-  def test_recipient_bcc
+  test "#recipient_bcc raises on an invalid email" do
     assert_raises(USPSValidationError) do
-      @external_request_label_req.recipient_bcc = "not_a_valid_email"
+      @external_request_label_req.recipient_bcc = @invalid_email
     end
-    assert_silent do
-      @external_request_label_req.recipient_bcc = "no-reply@example.com"
+  end
+
+  test "#recipient_bcc assigns the email" do
+    assert_nothing_raised do
```

Comment:
[`assert_nothing_raised` is pretty useless at best and harmful at worst](http://www.thagomizer.com/blog/2013/06/11/assert-nothing-raised.html), since all Minitest test cases are expected to not raise.